src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-pop-up/ubs-admin-tariffs-location-pop-up.component.html

```diff
@@ -25,28 +24,23 @@
           />
           <div *ngIf="data.edit">
             <input
-            type="text"
-            class="shadow-none form-control"
-            required
-            matInput
-            formControlName="region"
-            placeholder="Обрати область"
-            [matAutocomplete] = "autoRegion"
-            #autocompleteTriggerRegion="matAutocompleteTrigger" 
+              type="text"
+              class="shadow-none form-control"
+              required
+              matInput
+              formControlName="region"
+              placeholder="Обрати область"
```

Comment:
Shouldn't the placeholder be translated?

Reply:
It is translated.Honestly, idk why it is shown like this
![image](https://user-images.githubusercontent.com/86615598/193514764-1258a570-a28e-4284-bfee-748ac970ae53.png)